kin-base/src/main/java/kin/base/Server.java

```diff
@@ -178,6 +183,26 @@ public SubmitTransactionResponse submitTransaction(Transaction transaction) thro
         return null;
     }
 
+    /**
+     * Submits transaction to the network.
+     *
+     * @param transaction transaction to submit to the network.
+     * @return {@link SubmitTransactionResponse}
+     */
+    public SubmitTransactionResponse submitTransaction(Transaction transaction) throws IOException {
+        return submitTransaction(transaction, null);
+    }
+
+    private Uri getUri(String url) {
```

Comment:
Missing unit test :(

Reply:
I wrote that i still need to add unit test but firstly wanted to start the OR